net/strongswan/files/ipsec.init

```diff
@@ -267,24 +296,59 @@ config_remote() {
 	ipsec_xappend ""
 }
 
-config_ipsec() {
-	local debug
-	local rtinstall_enabled
-	local routing_tables_ignored
-	local routing_table
-	local routing_table_id
-	local interface
-	local device_list
-
+do_resets() {
 	ipsec_reset
 	secret_reset
 	swan_reset
+}
 
+do_preamble() {
 	ipsec_xappend "# generated by /etc/init.d/ipsec"
 	ipsec_xappend "version 2"
 	ipsec_xappend ""
 
 	secret_xappend "# generated by /etc/init.d/ipsec"
+}
+
+config_setup() {
+	local cachecrls
+	local charondebug
+	local strictcrlpolicy
+	local uniqueids
+
+	[ -n "$seen_setup" ] && return
```

Comment:
Doesn't make sense to have multiple such sections but there's no built-in notion of singular sections in UCI.